samples/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(NOVELRT_SAMPLES_HEADERS)
+source_group(TREE ${CMAKE_SOURCE_DIR} FILES ${NOVELRT_SAMPLES_HEADERS})
+
+file(GLOB_RECURSE NOVELRT_SAMPLES_SOURCES CONFIGURE_DEPENDS *.cpp)
```

Comment:
@RubyNova, I know you said you had problems with `GLOB`; but from what I can tell `CONFIGURE_DEPENDS` ensures that the configuration is updated if the glob changes (i.e. if you add a new file to the disk).